content/install_maintain/installation/installing-cluster.md

```diff
@@ -129,9 +117,16 @@ The following sections describe how to install and configure Cloudify Manager cl
 1. [PostgresSQL Database Cluster ] ({{< relref "install_maintain/installation/installing-cluster.md#postgresql-database-cluster" >}})  
 2. [RabbitMQ Cluster] ({{< relref "install_maintain/installation/installing-cluster.md#rabbitmq-cluster" >}})  
 3. [Cloudify Management Service] ({{< relref "install_maintain/installation/installing-cluster.md#cloudify-management-service" >}}) 
-4. [Management Service Load Balancer] ({{< relref "install_maintain/installation/installing-cluster.md#management-service-load-balancer" >}}) 
+4. [File Permissions]({{< relref "install_maintain/installation/installing-cluster.md#file-permissions" >}})
+5. [Management Service Load Balancer] ({{< relref "install_maintain/installation/installing-cluster.md#management-service-load-balancer" >}}) 
+
+### Preperation
+1. Ensure you have nine VMs with cfy_manager available on each. 
+1. All VMs should be on the same network and there should be no firewall/security group blocking any of our services.
+1. For each instance, please copy cloudify license to home directory.   
+1. Copy the /home/centos/.cloudify-test-ca directory from the VM where you generated the certs to the same location on both the other VMs.
```

Comment:
Agreed, this should refer to "the other VMs" and can note that each instance only needs its own certs+keys.

In fact, for production use it would be best practise for the keys for each host to never be seen by any other host.

Also, it should be noted that the .cloudify-test-ca directory approach to this only applies when using cfy_manager generate-test-cert for example/test certificates. If another, e.g. company, CA was used then the relevant certs will still need copying, but they won't come from the .cloudify-test-ca directory.

Reply:
done